### PR TITLE
[DROOLS-4332] - Disable PMML parsing via kie-pmml if jPMML is in the classpath

### PR DESCRIPTION
**Pull request by vblagoje**
See https://issues.jboss.org/browse/DROOLS-4332 for more details. @danielezonca should review again. More reviews welcome.

### CODE REVIEW

**Review by danielezonca — APPROVED** [review]

**Comment by danielezonca** [comment]
@kkufova @jiripetrlik @jomarko 
Can you please have a look to this PR?

**Comment by kkufova** [comment]
I don't have enough time to check this. I'm leaving the review to @jiripetrlik and @jomarko.

File: drools-core/src/main/java/org/drools/core/command/runtime/pmml/ApplyPmmlModelCommand.java [hunk 1]
```diff
@@ -290,5 +293,14 @@ public PMML4Result execute(Context context) {
         return resultHolder;
     }
 
+    private boolean isjPMMLAvailableToClassLoader(ClassLoader classLoader) {
+        try {
+            classLoader.loadClass("org.kie.dmn.jpmml.DMNjPMMLInvocationEvaluator");
+            return true;
+        } catch (ClassNotFoundException e) {
```
Comment (jiripetrlik) [inline drools-core/src/main/java/org/drools/core/command/runtime/pmml/ApplyPmmlModelCommand.java:300]: Please handle this exception. Either rethrow or handle other way.

File: kie-pmml/src/main/java/org/kie/pmml/assembler/PMMLAssemblerService.java [hunk 2]
```diff
@@ -291,4 +298,12 @@ private JavaCompiler createCompiler(JavaDialectConfiguration javaConf, String pr
         return javaCompiler;
     }
 
+    private boolean isjPMMLAvailableToClassLoader(ClassLoader classLoader) {
+        try {
+            classLoader.loadClass("org.kie.dmn.jpmml.DMNjPMMLInvocationEvaluator");
+            return true;
+        } catch (ClassNotFoundException e) {
```
Comment (jiripetrlik) [inline kie-pmml/src/main/java/org/kie/pmml/assembler/PMMLAssemblerService.java:305]: Please handle this exception. Either rethrow or handle other way.

**Review by jiripetrlik — CHANGES_REQUESTED** [review]
Two minor comments. But need to test locally before merge.
@vblagoje Any chance to add unit test to PR?

File: drools-core/src/main/java/org/drools/core/command/runtime/pmml/ApplyPmmlModelCommand.java (hunk 1 above)
Comment (jiripetrlik) [reply drools-core/src/main/java/org/drools/core/command/runtime/pmml/ApplyPmmlModelCommand.java:300]: Or it is also possible to move "return false" to catch block.

File: kie-pmml/src/main/java/org/kie/pmml/assembler/PMMLAssemblerService.java (hunk 2 above)
Comment (jiripetrlik) [reply kie-pmml/src/main/java/org/kie/pmml/assembler/PMMLAssemblerService.java:305]: Or it is also possible to move "return false" to catch block.

**Comment by danielezonca** [comment]
jenkins execute compile downstream build

**Review by jiripetrlik — APPROVED** [review]

**Pull request merged**